src/OrleansAWSUtils/Streams/SQSBatchContainer.cs

```diff
@@ -0,0 +1,112 @@
+﻿using Amazon.SQS.Model;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Orleans.Providers.Streams.Common;
+using Orleans.Runtime;
+using Orleans.Serialization;
+using Orleans.Streams;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using SQSMessage = Amazon.SQS.Model.Message;
+
+namespace OrleansAWSUtils.Streams
+{
+    [Serializable]
+    internal class SQSBatchContainer : IBatchContainer
```

Comment:
Agreed.  Duplicate code sucks, but building a common batch container base class can easily be argued as out of scope for this PR.